apidoc2swagger/apidoc2swagger.php

```diff
@@ -72,14 +73,37 @@ public function convert($_apiDocFileName = "FileCloud-apidoc.json", $_swaggerFil
                         $required = true;
                         if (strpos(strtolower($parameter["description"]), "(optional)") !== false) $required = false;
 
+                        $in = "";
+                        if (strtolower($apiCall["type"]) == "get") $in = "query";
+                        else if (strtolower($apiCall["type"]) == "post" && $apiCall["url"] != "/core/upload")
+                        {
+                            $in = "formData";
+                            $pathArray->$requestType->consumes = array("application/x-www-form-urlencoded");
+                        }
+                        else if (strtolower($apiCall["type"]) == "post" && $apiCall["url"] == "/core/upload")
+                        {
+                            $in = "query";
+                            $pathArray->$requestType->consumes = array("multipart/form-data");
+                        }
+
                         array_push($pathArray->$requestType->parameters, (object)array(
                             "name" => $parameter["field"],
-                            "in" => "query",
+                            "in" => $in,
                             "type" => $type,
                             "description" => trim(strip_tags($parameter["description"])),
                             "required" => $required
                         ));
                     }
+                    if ($apiCall["url"] == "/core/upload")
+                    {
+                        array_push($pathArray->$requestType->parameters, (object)array(
+                            "name" => "file_contents",
+                            "in" => "formData",
+                            "type" => "file",
+                            "description" => "The path to the file you want to upload.",
```

Comment:
Really the path? I thought this should be the contents? Or does the generated swagger classes automatically load the data from the provided path?